docs/_templates/class.rst

```diff
@@ -0,0 +1,13 @@
+:mod:`{{module}}`.{{objname}}
+{{ underline }}==============
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+    :members:
+    :special-members:
+    :exclude-members: __init__
+
+.. raw:: html
+
+    <div class="clearer"></div>
```

Comment:
why did this need? looks strange, probably this line should be in template of site, not `rst` template

Reply:
Well, I copypasted these templates from sklearn. :)

Reply:
It's working, isn't it? We'll remove redundant parts later.

Reply:
If this class isn't defined - this doesn't work. Try to disable & build & compare please

Reply:
Ok, will do

Reply:
https://forum.dokuwiki.org/thread/10120